bookkeeper-common/pom.xml

```diff
@@ -30,6 +30,11 @@
       <artifactId>bookkeeper-stats-api</artifactId>
       <version>${project.parent.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.apache.bookkeeper</groupId>
```

Comment:
since you are adding a dependency in bookkeeper-common, can you update the shading rules ?

https://github.com/apache/bookkeeper/blob/master/shaded/bookkeeper-server-shaded/pom.xml#L68

https://github.com/apache/bookkeeper/blob/master/shaded/distributedlog-core-shaded/pom.xml#L89

Reply:
Fixed